app/views/invitations/index.html.erb

```diff
@@ -1,11 +1,11 @@
 <!--Descriptive page name, messages and instructions-->
```

Comment:
Thanks!!! That was super helpful! I've pushed a few updates that should resolve the issue.